types/reactstrap/lib/CardLink.d.ts

```diff
@@ -2,7 +2,7 @@ import { CSSModule } from '../index';
 
 export interface CardLinkProps extends React.HTMLAttributes<HTMLElement> {
   tag?: React.ReactType;
-  innerRef?: string | ((instance: HTMLButtonElement) => any);
+  innerRef?: React.Ref<HTMLAnchorElement>;
```

Comment:
Not sure why this was originally typed with `HTMLButtonElement`; the default behavior is to create `<a>`s (oversight?).